pkg/app/piped/planner/kubernetes/kubernetes.go

```diff
@@ -200,7 +200,12 @@ func (p *Planner) Plan(ctx context.Context, in planner.Input) (out planner.Outpu
 			return
 		}
 
-		loader := provider.NewLoader(in.ApplicationName, runningDs.AppDir, runningDs.RepoDir, in.GitPath.ConfigFilename, cfg.Input, in.GitClient, in.Logger)
+		runningCfg := runningDs.ApplicationConfig.KubernetesApplicationSpec
+		if runningCfg == nil {
```

Comment:
Sorry, I couldn't understand this meaning.
What happens with the first plan?

Reply:
cfg.Input is the application configuration of the target commit (the one which we're going to deploy), but at that place what we need is the currently running application configuration, that is why I pull runningCfg there

ref: https://github.com/pipe-cd/pipecd/blob/master/pkg/app/piped/planner/kubernetes/kubernetes.go#L56-L65

Reply:
Oh, I totally get it.
I misunderstood the application manifest and application config.
Thank you.